content/wiki/organising-icpc-contest.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Organising an ICPC style contest"
+date: 2022-04-18T11:50:27+02:00
+---
+
+One of the first activities a new organisation can try is to host an event for first-years to introduce them to Competive Programming. Currently the universities organise this independently but there have been years where problem sets have been shared. If this is the first time that you want to organise something like this feel free to tag people in the [Slack](/Benelux-SLACK.md) and ask for help.
+
+The general information (although very focused on the style chosen in the USA) can be found here: https://icpc.global/regionals/regional-contest-cookbook. In the BAPC region, DOMjudge is chosen more often, and for your first contest it is a good idea to ask if there is anyone who still has an instance available to make focusing on the contest easier. This is not because setting up the technical part is hard but more because a lot can go wrong in setting up your contest and outsourcing as much as possible is much easier to focus on all the things you might not have planned for.
+
+Over the years, we have learned the following things. Listed below are common pitfalls and must-haves. Note, this is a work-in-progress which can and should be extended.
+
+### General
+- The use of radios is strongly adviced. Be sure that key persons have one and are available or tell who will take over when they are e.g. giving a presentation.
+- Have a dedicated photographer who knows what they are doing.
+- Not all teams have participated in a programming contest before.
+- Dinner between the contest and the award ceremony is greatly appreciated by contestants.
+- Have a dedicated contact person for companies during the contest day.
+- Have enough runners available, especially after the contest. Have an organisation person oversee the runners and check that they stay at their posts.
+- Be sure to have enough time to deal with the ICPC system.
+- Do everything you can do before the contest day before the contest day.
+- Use your preliminaries as a test for the BAPC for both systems and logistics.
+
+### Jury
+- There should be one person that does the communication between jury and organisation.
+
+### System
+- Avoid using Wi-Fi as networking solution.
+
+### Preliminaries
+- Promote preliminary contests to use the latest versions of DOMjudge.
```

Comment:
I think 1 version is fine, I don't see the benefit in hosting multiple instances
```suggestion
- Promote preliminary contests to use the latest version of DOMjudge.
```